Sources/Testing/ABI/ABI.swift

```diff
@@ -54,28 +81,28 @@ extension ABI {
   ///
   /// - Warning: This type will be removed in a future update.
   enum Xcode16: Sendable, Version {
-    static var versionNumber: Int {
-      -1
+    static var versionNumber: VersionNumber {
+      VersionNumber(-1, 0)
     }
   }
 #endif
 
   /// A namespace and type for ABI version 0 symbols.
   public enum v0: Sendable, Version {
-    static var versionNumber: Int {
-      0
+    static var versionNumber: VersionNumber {
+      VersionNumber(0, 0)
     }
   }
 
-  /// A namespace and type for ABI version 1 symbols.
+  /// A namespace and type for ABI version 6.3 symbols.
   ///
   /// @Metadata {
-  ///   @Available("Swift Testing ABI", introduced: 1)
+  ///   @Available(Swift, introduced: 6.3)
   /// }
   @_spi(Experimental)
-  public enum v1: Sendable, Version {
-    static var versionNumber: Int {
-      1
+  public enum v6_3: Sendable, Version {
+    static var versionNumber: VersionNumber {
+      VersionNumber(6, 3)
```

Comment:
These were the `init` usages which made me suggest having explicit `major: ...` labels, to clarify what the numbers meant.

Reply:
This is, again, just for convenience. If it really bothers you I can remove the convenience initializer.

Reply:
Made the convenience initializer take `_const` values so that you can only use it with integer literals so it's always clear at the call site.